packages/legend-extension-dsl-persistence/src/graph/metamodel/pure/model/packageableElements/persistence/DSL_Persistence_Deduplication.ts

```diff
@@ -36,8 +37,12 @@ export class AnyVersion extends Deduplication {
 export abstract class MaxVersion extends Deduplication {}
 
 export class MaxVersionForGraphFetch extends MaxVersion {
+  versionFieldPath!: object;
```

Comment:
same comment, please annotate this discrepancy

Reply:
Updated.